plugins/cost-insights/src/components/CostOverviewCard/CostOverviewCard.test.tsx

```diff
@@ -38,13 +39,30 @@ const mockGroupDailyCost: Cost = {
   trendline: trendlineOf(MockAggregatedDailyCosts),
 };
 
+type TestInputPluginOptions = {
+  showTrendLine: boolean;
+};
+
+type TestPluginOptions = {
+  showTrendLine: boolean;
+};
+
+const plugin = createPlugin({
```

Comment:
We probably need some form of test util to reduce the boilerplate to test this functionality, probably good to add that before as a separate PR.

Reply:
Added the util method in this PR, as it's pretty small and quite fit to this PR as well 😅